README.adoc

```diff
@@ -112,47 +128,116 @@ NOTE: The `+"auth":"<username>:<password>"+` field must be base64-encoded.
 
 | `artifactIds`
 | String
-| The output parameter for artifact IDs.
+| The unique identifier of the artifact reported to the CloudBees platform.
+|===
 
-These artifact IDs values can be accessed in subsequent steps using the outputs context. For example, to return:
+=== Using the action output
```

Comment:
IMO "Using the action output" section should go under the "Usage examples" section possibly after the "Basic example" or even at the end of the examples section.